model/http-common.yaml

```diff
@@ -63,6 +63,32 @@ groups:
           conditionally_required: If and only if one was received/sent.
         brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
         examples: [200]
+      - id: request.error.reason
+        type:
+          allow_custom_values: true
+          members:
+            - id: timeout
+              value: "timeout"
+              brief: 'The request timed out.'
+            - id: canceled
+              value: "canceled"
+              brief: 'The request was canceled.'
```

Comment:
👍

Reply:
@mateuszrzeszutek thanks! I just removed those based on @antonfirsov's feedback and to avoid controversy. 

LMK if you see value in bringing those back :)